CHANGELOG.md

```diff
@@ -5,9 +5,13 @@
 - This project uses [ISO Standard](https://www.iso.org/iso-8601-date-and-time-format.html) date formatting
 
 ## [Unreleased]
+### Added
+- Interaction Events.
+
 ### Fixed
 - macOS Sonoma crashing the application by updating dependencies.
 - Missing File Finder thinking assets folder doesn't exist for release builds.
+- Movement no long stutters.
```

Comment:
It still happens unfortunately :c 

I'd recommend making a new issue to investigate the movement stuttering on its own.

Reply:
My disappointment is immeasurable and my day is ruined. I'll remove it